modules/core/shared/src/main/scala/weaver/Result.scala

```diff
@@ -164,38 +177,19 @@ object Result {
       color: String,
       prefix: String): String = {
 
-    val footer = locationFooter(location)
-    val lines = (message.split("\\r?\\n") ++ footer).zipWithIndex.map {
-      case (line, index) =>
-        if (index == 0)
-          color + prefix + line +
-            location
-              .map(l => s" (${l.fileRelativePath}:${l.line})")
-              .mkString("\n")
-        else
-          color + prefix + line
-    }
-
-    lines.mkString(EOL) + Console.RESET
-  }
-
-  private def indent(
-      message: String,
```

Comment:
The `indent` function has been removed, and its usages have been replaced with `formatDescription`.

The functions were practically identical. The only difference was the treatment of the `prefix`. 
 - `indent` would assume the prefix was whitespace, and so not add it for empty lines.
 - `formatDescription` would always add the prefix

This change rewrites `formatDescription` to test if the prefix is whitespace and line is empty, and if so not add the prefix.